### DISCUSSION TITLE
give a notification an experation

### DISCUSSION THREAD

**Discussion by TD-4242** [magicbell/community#202] [General]
We would like to have a notification of system maintenance, but would like to be able to expire it once the maintenance is over so anyone that isn't affected by the window isn't bothered by the notification.

I don't see any way in current notifications, nore do I see a way to manually remove a mass notification.

**Comment by unamashana** [comment]
@TD-4242 Thanks for the suggestion and apologies for missing this. We plan to add this to MagicBell. Unfortunately, we don't have an ETA for this yet.
